File: 1100/224A/main.java
import java.util.*;
public class main
{
    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        int b = sc.nextInt();
        int c = sc.nextInt();
        
        int k = (int)Math.sqrt((a*c)/b);
        
        int side1 = k;
        int side2 = a/k;
        int side3 = (b*k)/a;
        
        System.out.println(4*(side1 + side2 + side3));
    }
}

File: 1100/224A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() 
{
    int a, b, c;
    cin >> a >> b >> c;

    int k = (int) sqrt((1.0 * a * c) / b);

    int side1 = k;
    int side2 = a / k;
    int side3 = (b * k) / a;

    cout << 4 * (side1 + side2 + side3) << "\n";

    return 0;
}
